### ISSUE TITLE
`pip install nupic` error: "clang: error: no input files" [OS X 10.9]

### ISSUE THREAD

**Issue by rhyolight** [numenta/nupic#1755]
[Console log](http://pastebin.com/TPCn8zJr).

```
building 'fake-extension' extension

clang -bundle -undefined dynamic_lookup -L/usr/local/lib -L/usr/local/opt/sqlite/lib -Wno-error=unused-command-line-argument-hard-error-in-future -o build/lib.macosx-10.9-x86_64-2.7/fake-extension.so

clang: error: no input files

error: command 'clang' failed with exit status 1
```

http://lists.numenta.org/pipermail/nupic_lists.numenta.org/2015-January/010148.html

**Comment by rhyolight** [comment]
The console output indicates that the nupic.core binary was downloaded twice! :confounded:

**Comment by hstm** [comment]
Had the same issue during a developer installation (python setup.py develop). Seems to be fixed by removing the fake c extension (#1770).

**Comment by rhyolight** [comment]
@hstm Yes, thank you. I think I can close this now.
